R/model_component_fwd_sim.R

```diff
@@ -0,0 +1,472 @@
+#' Function to generate a global weekly R(t) estimate with added noise
+#'
+#' @param r_in_weeks The mean R(t) for each week of the simulation, can be
+#' longer than the 7*n_weeks
+#' @param n_weeks The number of weeks that we want to simulate the data for
+#' @param global_rt_sd The variation in the R(t) estimate to add
+#' intrinsic variability to the infection dynamics
+#'
+#' @return a weekly R(t) estimate with added noise
```

Comment:
Since all these functions are associated, you can use the [`family` roxygen tag](https://roxygen2.r-lib.org/articles/index-crossref.html#family) to cross-reference them automatically. 

```suggestion
#' @family Model components
#' @return a weekly R(t) estimate with added noise
```

Reply:
Magic! So helpful thank you